cpp/src/stream_compaction/drop_duplicates.cu

```diff
@@ -133,14 +133,16 @@ column_view get_unique_ordered_indices(cudf::table_view const& keys,
                                        cudf::mutable_column_view& unique_indices,
                                        duplicate_keep_option keep,
                                        null_equality nulls_equal,
+                                       null_order null_precedence,
                                        rmm::cuda_stream_view stream)
 {
   // sort only indices
-  auto sorted_indices = sorted_order(keys,
-                                     std::vector<order>{},
-                                     std::vector<null_order>{},
-                                     stream,
-                                     rmm::mr::get_current_device_resource());
+  auto sorted_indices = sorted_order(
+    keys,
+    std::vector<order>{},
+    std::vector<null_order>{static_cast<uint64_t>(keys.num_columns()), null_precedence},
```

Comment:
technically this could create a bottleneck if `num_columns` is pretty high, but that's a design decision of sorted_order and addressing it is not in the scope of these changes imho.

Reply:
is this static cast necessary?

Reply:
its otherwise a warning. num_columns is signed, whereas the length parameter is unsigned.